.github/workflows/check-format.yml

```diff
@@ -1,4 +1,4 @@
-name: Test the automatic code formatting task
+name: Check if all files are formatted correctly with spotless
```

Comment:
```suggestion
name: Spotless check
```

Reply:
Note that this is rendered in the left-hand menu, and if it is too long it just gets cut off, so it's better to keep this short.

Reply:
:rofl: @petervdonovan you just merged this. Could you apply this change in `master`? Thanks!